dhis-2/dhis-api/src/main/java/org/hisp/dhis/scheduling/JobConfiguration.java

```diff
@@ -38,17 +38,7 @@
 import org.hisp.dhis.common.BaseIdentifiableObject;
 import org.hisp.dhis.common.DxfNamespaces;
 import org.hisp.dhis.common.SecondaryMetadataObject;
-import org.hisp.dhis.scheduling.parameters.AnalyticsJobParameters;
-import org.hisp.dhis.scheduling.parameters.ContinuousAnalyticsJobParameters;
-import org.hisp.dhis.scheduling.parameters.DataSynchronizationJobParameters;
-import org.hisp.dhis.scheduling.parameters.DisableInactiveUsersJobParameters;
-import org.hisp.dhis.scheduling.parameters.EventProgramsDataSynchronizationJobParameters;
-import org.hisp.dhis.scheduling.parameters.MetadataSyncJobParameters;
-import org.hisp.dhis.scheduling.parameters.MonitoringJobParameters;
-import org.hisp.dhis.scheduling.parameters.PredictorJobParameters;
-import org.hisp.dhis.scheduling.parameters.PushAnalysisJobParameters;
-import org.hisp.dhis.scheduling.parameters.SmsJobParameters;
-import org.hisp.dhis.scheduling.parameters.TrackerProgramsDataSynchronizationJobParameters;
+import org.hisp.dhis.scheduling.parameters.*;
```

Comment:
I am not sure if there was any strict rule on that. But I can change this to be safe anyway!

Reply:
Intellij automaticaally converts into a * as we are importing everything from thatt package anyway 🤔  I guess that is fine then?

Reply:
You need to configure it https://stackoverflow.com/questions/3587071/disable-intellij-starred-package-imports
It cannot be disabled but somehow tricked with saying only use `*` with 1000 imports 😂

Reply:
Ah. Wondering if it is really required in this case though. We definitely have to import everything from that package for every new job parameter we create anyway. So I feel it is okay to have the * here. What do you think?

Reply:
This was a style that stuck with me from other projects. I thought we adhere to it here too. Since we don't seem to and I haven't read up all the smart arguments against `*` (cluttered namespace is at least one that comes to mind) I'll resolve this one 😂

Reply:
Maybe for the next PRs 😄  According to Stian we avoid `*` imports.